build.sh

```diff
@@ -90,7 +90,6 @@ while [ $TRIES -gt 0 ]; do #try again once
         BUILD_ENGINE_INSTALLED= \
         BUILD_HE_INSTALLED= \
         OPENSCAP_PROFILE="${OPENSCAP_PROFILE}" \
-        USE_FIPS= \
```

Comment:
Just asking - why not explicitly enable FIPS? It'd be easier to disable if needed and easier to diagnose (is it on by default when you remove USE_FIPS with an empty value?)

Reply:
We could add these vars explictly for each of the conditionals in build.sh, but yes, FIPS is desired to be on by default and expected to work. This is determined by the var here https://github.com/oVirt/ost-images/blob/b7f9960bfc34305f5b3c8c21fb1c0ace705c98b8/makefiles/vars.mk#L26 and the command line option passed here https://github.com/oVirt/ost-images/blob/b7f9960bfc34305f5b3c8c21fb1c0ace705c98b8/makefiles/built-base.mk#L43

We've made a temporary exception to el9stream and the 'USE_FIPS' var was added specifically to handle this situation.